laser_control/laser_control/nodes/laser_control_node.py

```diff
@@ -73,7 +77,7 @@ def __init__(self):
             dac.name = f"Ether Dream {i}"
             self.dac_list.append(dac)
 
-        self.logger.info(
+        self.get_logger().info(
```

Comment:
super super minor but since get_logger() is already provided to us and this is the only place it's used here, I figured might as well use it instead of adding another logger field.